api_app/visualizers_manager/classes.py

```diff
@@ -379,3 +379,35 @@ def connector_reports(self) -> QuerySet:
         if configs:
             queryset = queryset.filter(config__in=configs)
         return queryset
+
+
+# IMPORTANT! this function allows to handle the errors in the components render.
+# You must define a function that returns a Visualizable then
+# use this function as a decorator. Ex:
+# @visualizable_error_handler_with_params("field1", VisualizableSize.S_2)
+# def generate_field1(value):
+# ...
+# in case the generation of the field will raise an error
+# this will handle it allowing to render the other components
```

Comment:
I would like you to add something regarding this in the official docs where we explain how to write a visualizer too